pkg/operation/botanist/secrets.go

```diff
@@ -232,6 +232,10 @@ func (b *Botanist) rotateKubeconfigSecrets(ctx context.Context, gardenerResource
 		secrets = append(secrets, konnectivity.SecretNameServerKubeconfig)
 	}
 
+	if b.isShootNodeLoggingActivated() {
```

Comment:
Enabled instead of activated sounds better to me. If you also think so, you can change it.

Reply:
Ok you are right.